packages/side-nav/README.md

```diff
@@ -173,6 +173,7 @@ _Any other properties will be spread on the root-level element._
 | `children`         | `node`           | Content that will be rendered inside the root-level element.                                                                                     | `undefined` |
 | `collapsible`      | `boolean`        | Determines whether or not the Group is collapsible                                                                                               | `false`     |
 | `initialCollapsed` | `boolean`        | Determines whether or not the SideNavGroup should be collapsed on the first render.                                                              | `true`      |
+| `hasActiveItem`    | `boolean`        | Manually overrides automatic detection of whether a group contains an active item when the nav is collapsed.                                     |             |
```

Comment:
This language is confusing. Right now the only implication is styles applied when the nav is collapsed, but I don't think our implementation is limited to this (in the case that we want to use `isActiveGroup` for other logical decisions)